### ISSUE TITLE
[BUG] Form crashes in "viewAsHtml" mode

### ISSUE THREAD

**Issue by vanekt** [formio/formio.js#1460]
Hi! 

## Environment

- Formio.js version: 3.23.0 (reproduced on 3.21.0+)
- Frontend framework: No framework
- Browser: Google Chrome
- Browser version: 75.0.3770.100

## Steps to Reproduce
If form contains the checkbox component, it crashes in "viewAsHtml" mode. 

```js
Uncaught (in promise) TypeError: Cannot read property 'checked' of undefined
    at CheckBoxComponent.updateValue (Checkbox.js:542)
    at CheckBoxComponent.setValue (Checkbox.js:505)
    at ColumnComponent.setNestedValue (NestedComponent.js:895)
    at eval (NestedComponent.js:912)
    at Array.reduce (<anonymous>)
    at ColumnComponent.setValue (NestedComponent.js:911)
    at ColumnsComponent.setNestedValue (NestedComponent.js:893)
    at eval (NestedComponent.js:912)
    at Array.reduce (<anonymous>)
    at ColumnsComponent.setValue (NestedComponent.js:911)
    at PanelComponent.setNestedValue (NestedComponent.js:893)
    at eval (NestedComponent.js:912)
    at Array.reduce (<anonymous>)
    at PanelComponent.setValue (NestedComponent.js:911)
    at TabsComponent.setNestedValue (NestedComponent.js:893)
    at eval (NestedComponent.js:912)
    at Array.reduce (<anonymous>)
    at TabsComponent.setValue (NestedComponent.js:911)
    at Webform.setNestedValue (NestedComponent.js:893)
    at eval (NestedComponent.js:912)
    at Array.reduce (<anonymous>)
    at Webform.setValue (NestedComponent.js:911)
    at Webform.setValue (Webform.js:976)
    at Webform.setSubmissionValue (Webform.js:830)
    at eval (Webform.js:866)
```

**Comment by wag110894** [comment]
Hello, Thanks for reaching out!

We have added this issue to our backlog. We will let you know once this issue is resolved.

**Comment by jhen1422** [comment]
This issue should be resolved in version 3.23.3
